src/components/layout/PrimaryLayout/PrimaryLayout.js

```diff
@@ -19,7 +22,23 @@ export function PrimaryLayout ({
   showSideMenu = true,
   showAlphaFlag = true
 }) {
-  const { data: { hposSettings: settings = {} } = {} } = useQuery(HposSettingsQuery)
+  const { setIsConnected, isConnected } = useConnectionContext()
+
+  const onError = ({ graphQLErrors }) => {
+    const hposConnection = graphQLErrors.hposConnection
```

Comment:
nbd but you could also write this as 
`const { hposConnection } = graphQLErrors`